src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,514 @@
+	"AttenuationCorrection": "[137Cs]transmission scan-based"
+}
+```
+
+## Blood recording data
+
+This section includes all data needed to perform blood analysis for PET data.
+The section may be omitted completely if blood measurements of radioactivity were not made.
+If blood measurements were made a `*_blood.json` file is expected that contains
+information about which blood data is available as well as some blood measurement details.
+Furthermore, depending on the information given in the `*_blood.json` file
+we expect `.tsv` files with corresponding .json files that contain the values
+and information regarding plasma samples, metabolite fractions,
+discrete blood samples and/or continuous blood samples from an autosampler.
+All these measurements should be defined according to a single time-scale
+in relation to time zero defined by the PET data (Figure 1).
+Additionally, if blood and metabolite measurements were made
+one should always report radioactivity in plasma and corresponding
+parent fraction measurements, including fractions of radiometabolites.
+All definitions used below are in accordance with Innis et al. 2007 [3].
+All method specific information related to the measurements can be stated
+in the field `Description`.
+
+Template:
+
+```Text
+sub-<participant_label>/
+ 	[ses-<session_label>/]
+	pet/
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_blood.json
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>.tsv
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>.json
+
+```
+
+Blood data belongs in the /pet folder along with the corresponding PET data.
+However, the blood data also follows the [inheritance principle](../02-common-principles.md#the-inheritance-principle) and may be moved to an upper level folder
+if it does not change, for example, with multiple reconstructions.
+The blood data is most often recorded using an autosampler for continuous blood samples,
+or manually for discrete blood samples.
+Therefore, the recording key (`recording-<label>`) has two reserved values:
+`blood_continuous`,
+for continuous whole blood data measurements (2.2.5);
+and `blood_discrete`, for discrete blood data, including
+whole blood (2.2.5),
+plasma (2.2.3),
+parent fraction and
+metabolite fractions (2.2.4).
+The actual blood radioactivity data will be stored in [tabular `*.tsv` files](../02-common-principles.md#tabular-files)
+with the `blood_continuous` and `blood_discrete` suffix.
+The first column in the `*.tsv` file should be a time column (see 2.2.1),
+defined in relation to time zero defined by the `_pet.json` file.
+All other information relevant to the blood measurements are recommended,
+and can be added as an additional column.
+It is expected that all values are (if relevant) decay corrected to time zero.
+
```

Comment:
Add an example of TSV file

Reply:
I will add this in the text instead of as a figure.